source/slang/slang-emit-source-writer.cpp

```diff
@@ -224,7 +224,7 @@ void SourceWriter::emit(UInt64 value)
     emit(buffer);
 }
 
-void SourceWriter::emit(double value)
+inline static void SourceWriter_emit(SourceWriter *This, double value, int maxFracDigits)
```

Comment:
naming convention: we don't use underscores as decl names. This can be
`static void _emitFloatingPointLiteral()`